src/Linux/Avalonia.LinuxFramebuffer/Output/DrmPlatform.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Avalonia.OpenGL;
+using Avalonia.OpenGL.Egl;
+using Avalonia.OpenGL.Surfaces;
+using Avalonia.Platform.Interop;
+
+namespace Avalonia.LinuxFramebuffer.Output
+{
+    /// <summary>
+    ///     Provides the base handle to the DRM output and its associated EGL interface.
+    /// </summary>
+    /// <remarks>
+    ///     Shared resource creates a separation of the <see cref="IPlatformOpenGlInterface" /> instance, and
+    ///     the <see cref="DrmOutput" /> instances, which provide the <see cref="IGlPlatformSurface" /> for each DRM connector.
+    /// </remarks>
+    internal class DrmPlatform
+    {
+        public DrmPlatform(string path = null, double scaling = 1.0)
+            : this(new DrmCard(path), scaling)
+        {
+        }
+
+        public DrmPlatform(DrmCard card, double scaling = 1.0)
```

Comment:
Since you are adding multiple output support, scaling should be configured for each output separately

Reply:
Done.